### DISCUSSION TITLE
How to debug `Error: Resource not accessible by integration` errors

### DISCUSSION THREAD

**Discussion by kuisathaverat** [community/community#35213] [Actions]
I am starting to work with GitHub actions, and one of the recurrent issues I hit is the error `Error: Resource not accessible by integration`. Most of the time the action you are trying to use has some issue or some note that helps you to resolve the issue, but it is not ideal, and you have to find that issue or that note in the documentation. This task is time-consuming. 
So far all issues have been related to the permissions of the GITHUB_TOKEN, the resolution is to set the right permissions and that's it, but the error message does not give you any clue about the permission you need or the resource that is not accessible. It could be helpful to have the permission/resource fails to access into the message. Is there any way to debug these issues?

**Comment by ossanna16** [comment]
Great question @kuisathaverat! I'm glad you found our community! 🙂

**Comment by amitz-val** [comment]
Anything new on that matter?
